lib/scala/runtime-version-manager/src/main/java/module-info.java

```diff
@@ -1,6 +1,10 @@
 module org.enso.runtime.version.manager {
+  // Needed for `org.apache.tika.core` automatic module.
+  requires java.sql;
```

Comment:
- Adding SQL dependency just because of https://tika.apache.org/ is unfortunate
- unfortunate, but acceptable
- the alternative is to:
   - stop using the Tika API directly
   - use [FileTypeDetector](https://github.com/apache/tika/blob/1675fe22870d1a5679fa15248650569e3bd6ec52/tika-java7/src/main/resources/META-INF/services/java.nio.file.spi.FileTypeDetector#L1) which Tika provides
   - have a Tika wrapper which makes sure all that's needed is registered properly
- it would be nice cleanup